ros_installation.md

```diff
@@ -170,23 +170,43 @@ We do not recommend this approach for beginners!
 </div>
 
 To select a different release of Gazebo than the one officially supported by
-your ROS distribution you must **manually compile**
-[`ros_gz`](https://github.com/gazebosim/ros_gz) from source. This rule applies for
-every  ROS package that uses a Gazebo library. This approach may also require
-that you modify your  ROS or Gazebo source code to support this compilation.
+your ROS distribution you must either **use non ROS official Gazebo binary packages** or
+**manually compile**
+[`ros_gz`](https://github.com/gazebosim/ros_gz) from source (this rule applies for
+every ROS package that uses a Gazebo library):
+
+  * **Use non ROS official ros_gz binary packages:** *packages.osrfoundation.org* provides
+    binary packages for `ros_gz` for some combinations of Gazebo and ROS that are different
+    from the existing in ROS. In some circumstances these packages can be outdated or
+    even buggy, the use of them is reserved for advanced users.
+
+  * **Manually compile ros_gz:** some combinations of Gazebo and ROS can be prepared to be
+    built together but don't have official or unofficial binary packages.
+
+Both approaches may also require that you modify your ROS or Gazebo source code to support this compilation.
 
 ### Gazebo Garden (Not Recommended)
 
-Gazebo Garden can be used with ROS 2 Humble and ROS 2 Rolling, but
+Gazebo Garden can be used with ROS 2 Humble and non official ROS binary packages hosted
```

Comment:
To be consistent with above
"non ROS official"
Or maybe use hyphens to make it more clear
"non-ROS-official" everywhere?
Actually I see that a little below, it uses "unofficial." That's a shorter word, and I prefer it. Maybe just replace "non ROS official" and "non official ROS" with "unofficial" everywhere.
```suggestion
Gazebo Garden can be used with ROS 2 Humble and non ROS official binary packages hosted
```

Reply:
I concur, but "unofficial ROS" is the official replacement for "non-official ROS." 🫠